usagov/usagov-2021
# Usagov 1722 make usagov logo responsive

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-

## Description

Merge dev into branch, fixed bug of casing in image source in top_nav.html.twig

## Type of Changes

- [ x] Bugfix
  - Add screenshot showing what it should look like

## Testing Instructions



make sure usagov logo is still responsive like in prod and make sure "properly size image" and "explicit width and height" audits do not fail in lighthouse

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
